app/controllers/installer_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# InstallerController shows the main page.
+class InstallerController < ApplicationController
```

Comment:
Could you provide more helpful documentation ? This class has the same documentation as `ConfigController`. What's the difference between the two of them ? Could you write this on both classes ?